File: Modul-4/Soal-3/PRAK403-2410817220030-RACHELWINAYUDA.py
input_data = input()
angka1, angka2 = map(int, input_data.split())

if angka1 < angka2:
    for i in range(angka1, angka2 + 1):
        j = angka2 - (i - angka1)
        print(i, j, end="")
        if i < angka2:
            print(" - ", end="")
else:
    for i in range(angka1, angka2 - 1, -1):
        j = angka2 + (angka1 - i)
        print(i, j, end="")
        if i > angka2:
            print(" - ", end="")

File: Modul-4/Soal-3/PRAK403-2410817220030-RACHELWINAYUDA.c
#include <stdio.h>

int main() {
    int angka1, angka2, i, j;

    scanf("%d", &angka1);
    scanf("%d", &angka2);

    if (angka1 < angka2) {
        for (i = angka1, j = angka2; i <= angka2 && j >= angka1; i++, j--) {
            printf("%d %d", i, j);
            if (i < angka2) {
                printf(" - ");
            }
        }
    } else {
        for (i = angka1, j = angka2; i >= angka2 && j <= angka1; i--, j++) {
            printf("%d %d", i, j);
            if (i > angka2) {
                printf(" - ");
            }
        }
    }
    return 0;
}